com.napster.napster-bigscreen-electron.metainfo.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.napster.napster-bigscreen-electron</id>
+  <launchable type="desktop-id">com.napster.napster-bigscreen-electron.desktop</launchable>
+  <name>Napster for Steamdeck™</name>
```

Comment:
this is not what the upstream project says. Flathub isn't about Steamdeck.

Reply:
Agreed! Is that a no-go? Internally at Napster we're targeting Steamdeck, however, the app works just fine everywhere; most of the time we're just developing with manjaro/gnome.

Reply:
Also, later we'll probably submit a normal "Napster" app which will be the fully-fledged version (web version).
This is the lightweight client we've made for Xbox, LG/Samsung TVs, and Amazon Firestick (FireTV).

Reply:
Flathub is not SteamDeck-specific so yes, it should be removed.

Reply:
Changed to "Napster"

Reply:
If it is not the normal Napster, you should probably call it Napster Bigscreen or something else to avoid confusion

Reply:
It is Napster, and has all the features. Bigscreen is an internal name we use for the project and don't want to expose.

Reply:
I hate to break it to you, but it's exposed with the app-id. :-)

Reply:
indeed, what I meant is on the GUI of software stores like GNOME's/Discover. We're fine with packaging/repo/app-id. 👌